src/Aquifer.Common/Enums.cs

```diff
@@ -0,0 +1,6 @@
+namespace Aquifer.Common;
+
+public enum EmailTemplate
+{
+    AquiferMarketingNotification = 1
```

Comment:
This seems too vague to me. I would call it something like "MarketingNewContentNotification".